locales/en.default.schema.json

```diff
@@ -1010,15 +1022,6 @@
           "settings": {
             "heading": {
               "label": "Heading"
-            },
-            "heading_size": {
-              "options__1": {
-                "label": "Medium"
-              },
-              "options__2": {
-                "label": "Large"
-              },
-              "label": "Heading size"
```

Comment:
I think there are a few more you can remove no ? If I understand properly, we're using instead a general one instead of the same `"heading_size"` in all the section parts. 

I see 4 instances of `"heading_size"` but I believe we could have 1 only 🤔

Reply:
I only updated ones where I was already touching the file. 
I was planning to create a follow up PR where I could update the rest. Would that be okay with you?